vulture_os/services/frontend/config/haproxy_frontend.conf

```
@@ -89,9 +89,14 @@ cache cache_{{ conf.id }}
     {% endif -%}
 
     {% elif conf.mode in ["log", "filebeat"] -%} {# if conf.mode == "http" -#}
-        {% for listener in conf.listeners -%}
-            server server_{{ conf.name }}-{{loop.index}} {{listener.generate_server_conf()}}
-        {% endfor -%}
+    {% if conf.healthcheck_service and (conf.listening_mode in ["tcp", "tcp,udp", "relp"] or conf.filebeat_listening_mode in ["tcp"]) -%}
```

No need to specify those rules, you're already in the Haproxy configuration
```suggestion
    {% if conf.healthcheck_service -%}
```

---

Suggestion applied, thanks !